.gitignore

```diff
@@ -152,8 +152,13 @@ cython_debug/
 test_notebook.ipynb
 
 # saved bert model
+app/old_saved_model/
 app/saved_model/
+old_saved_model/
 saved_model/
 
 # scraper.log
-scraper.log
\ No newline at end of file
+scraper.log
+
```

Comment:
Just make sure to remember to remove any unused/old code before final PR.  i.e. double scraper.log and lines 155-158.  This just helps the next team working on this.